oct/ansible/oct/roles/repositories/tasks/main.yml

```diff
@@ -42,6 +42,7 @@
     - 'kubernetes-metrics-server'
     - 'online-console-extensions'
     - 'image-inspector'
+    - 'online-registration'
```

Comment:
Move up to the private repos block with `ose`

Reply:
done, thanks